app/models/collection_version.rb

```diff
@@ -14,65 +14,6 @@ def accessioned?
     %w[first_draft depositing].exclude?(state)
   end
 
```

Comment:
is this missing include statement?

Reply:
Sure looks like it.

Reply:
Oops, I forgot to add that change to the commit. Got it now.